### PR TITLE
Revises CL attributions; applies grammatical and Sphinx-syntax edits.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/clear-linux/tutorials/kubernetes.rst [hunk 1]
```diff
@@ -239,8 +240,8 @@ configuration files for each package.
```
Comment (krisdale) [inline source/clear-linux/tutorials/kubernetes.rst]: line 233: missed a place to use |CL|

File: source/clear-linux/tutorials/kubernetes.rst [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+Runc and Kata Containers\* kata-runtime adhere to :abbr:`OCI (Open Container Initiative*)` guidelines and work seamlessly with Kubernetes.
+`Kata Containers`_ provide strong isolation for untrusted workloads or 
+multi-tenant scenarios.  Runc and Kata Containers can be allocated on a 
+per-pod basis so you can mix and match both on the same host
+to suit your needs.
 
 Prerequisites
 *************
 
 This tutorial assumes you have installed |CL| updated to the latest release
```
Comment (krisdale) [inline source/clear-linux/tutorials/kubernetes.rst]: Line 23: "... installed CL and updated ..."

File: source/clear-linux/tutorials/kubernetes.rst [hunk 3]
```diff
@@ -239,8 +240,8 @@ configuration files for each package.
 
 If you customize any of the default package configuration files, you **must**
```
Comment (krisdale) [inline source/clear-linux/tutorials/kubernetes.rst]: Per the work-in-progress BKMs, we are using italic to emphasise text. So just a single *.

File: source/clear-linux/tutorials/kubernetes.rst [hunk 4]
```diff
@@ -251,7 +252,7 @@ following commands:
    sudo cp /usr/share/defaults/crio/crio.conf /etc/crio/
    sudo $EDITOR /etc/crio/crio.conf
 
-Learn more about `Stateless in Clear Linux`_ in the `Clear Linux documentation`_.
+Learn more about `Stateless in Clear Linux`_ and view the `Clear Linux documentation`_.
 
 Proxy configuration (optional)
 ******************************
```
Comment (krisdale) [inline source/clear-linux/tutorials/kubernetes.rst]: line 263: use of all caps for emphasis. Should instead use italics e.g. *explicitly included*

line 309: "verify what are the proxy" should be "verify the proxy"

line 313: "permanently and available from" should be "permanently, and make them available for"

**Comment by mvincerx** [comment]
@krisdale Thank you very much for the quick review. Auguri!

**Pull request merged**